protocols/jabber/block.c

```diff
@@ -0,0 +1,67 @@
+/***************************************************************************\
+*                                                                           *
+*  BitlBee - An IRC to IM gateway                                           *
+*  Jabber module - Handling of blocking                                     *
+*                                                                           *
+*  Copyright 2021 / <>                                                      *
```

Comment:
This should have a proper copyright statement

Reply:
how would that look like? we just copypasted it from some other file and replaced name/mail with our git details, thought that would be fine.

Reply:
This particular line is just empty, it should have your identity as the copyright owner.